outrank/core_ranking.py

```diff
@@ -115,8 +116,12 @@ def mixed_rank_graph(
     out_time_struct['encoding_columns'] = end_enc_timer - start_enc_timer
 
     combinations = get_combinations_from_columns(all_columns, args)
-    combinations = prior_combinations_sample(combinations, args)
-    random.shuffle(combinations)
+    #combinations = prior_combinations_sample(combinations, args)
```

Comment:
I am turning this feature off, since there is currently a bug in the interaction with line 216. I propose that we handle this separately.